### PR TITLE
[News] 2021-10-04

### PR DESCRIPTION
**Pull request by suetsuguk**
See also:
- https://github.com/sider/runners/blob/0.54.0/CHANGELOG.md
- sider/runners#2601
- sider/runners#2320

### CODE REVIEW

File: docs/news/2021.md [hunk 1]
```diff
@@ -10,6 +10,31 @@ This document describes notable changes on Sider in the year 2021.
...
+
+In addition, this release includes the following tool updates: ✨
+
+| Tool                                            | Language     | Version             | Ref                                                                           |
+| ----------------------------------------------- | ------------ | ------------------- | ----------------------------------------------------------------------------- |
+| [CoffeeLint](../tools/javascript/coffeelint.md) | CoffeeScript | 5.0.2 → **5.0.4**   | [changes](https://github.com/coffeelint/coffeelint/blob/v5.0.4/CHANGELOG.md)  |
+| [GolangCI-Lint](../tools/go/golangci-lint.md)   | Go           | 1.42.0 → **1.42.1** | [changes](https://github.com/golangci/golangci-lint/releases/tag/v1.42.1)     |
+| [Pylint](../tools/python/pylint.md)             | Python       | 2.10.2 → **2.11.1** | [changes](https://github.com/PyCQA/pylint/releases/tag/v2.11.1)               |
+| [remark-lint](../tools/markdown/remark-lint.md) | Markdown     | 9.0.0 → **10.0.0**  | [changes](https://github.com/remarkjs/remark/releases/tag/remark-cli%409.0.0) |
```
Comment (suetsuguk) [inline docs/news/2021.md:32]: I did not find a release tag for 10.0.0.

File: docs/tools/ruby/rails-best-practices.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
 ---
 id: rails-best-practices
-title: Rails Best Practices
-sidebar_label: Rails Best Practices (deprecated)
```
Comment (radomirbosak) [inline docs/tools/ruby/rails-best-practices.md:4]: sidebar_label is no longer needed?

Comment (suetsuguk) [reply docs/tools/ruby/rails-best-practices.md:4]: No. We will remove it from sidebar and can delete this line.

See also:
https://github.com/sider/sider-docs/pull/631/files#diff-a0383bb15ae59a60a14643df26cd68ac64da9837256a0e73b98cbad6b08397a3

File: docs/tools/ruby/rails-best-practices.md [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
+title: Rails Best Practices (unsupported)
 ---
 
-# Rails Best Practices
+# Rails Best Practices (unsupported)
 
 :::caution
```
Comment (dev-yakuza) [inline docs/tools/ruby/rails-best-practices.md:8]:
```suggestion
:::danger
```

How about using the same format?

- https://github.com/sider/sider-docs/blob/master/docs/tools/javascript/tslint.md

File: docs/tools/ruby/rails-best-practices.md [hunk 4]
```diff
@@ -1,16 +1,14 @@
 ---
 id: rails-best-practices
-title: Rails Best Practices
-sidebar_label: Rails Best Practices (deprecated)
+title: Rails Best Practices (unsupported)
 ---
 
-# Rails Best Practices
+# Rails Best Practices (unsupported)
```
Comment (dev-yakuza) [inline docs/tools/ruby/rails-best-practices.md:6]:
```suggestion
# Rails Best Practices
```

How about using the same format?

- https://github.com/sider/sider-docs/blob/master/docs/tools/javascript/tslint.md

File: docs/tools/ruby/rails-best-practices.md (hunk 3 above)
Comment (suetsuguk) [reply docs/tools/ruby/rails-best-practices.md]: Fixed. commit 98311d623ba1035b6ba142819700580672bf7ebf

File: docs/tools/ruby/rails-best-practices.md (hunk 4 above)
Comment (suetsuguk) [reply docs/tools/ruby/rails-best-practices.md]: Fixed. commit 98311d623ba1035b6ba142819700580672bf7ebf

File: docs/news/2021.md (hunk 1 above)
Comment (suetsuguk) [reply docs/news/2021.md:34]: Did anyone find a valid link? @dev-yakuza @radomirbosak

Comment (dev-yakuza) [reply docs/news/2021.md:34]: It's weird...I can't find it, too 🤔 

https://github.com/sider/runners/pull/2592

Comment (dev-yakuza) [reply docs/news/2021.md:34]: I made a discussion to ask about it.

- https://github.com/remarkjs/remark/discussions/868

Comment (dev-yakuza) [reply docs/news/2021.md:34]: The tag missing is not a mistake.

https://github.com/remarkjs/remark/discussions/868#discussioncomment-1407173

And I think they won't create a tag at this time. So, we need to use another link like the below for this time.

https://github.com/remarkjs/remark/releases/tag/14.0.0

Comment (radomirbosak) [reply docs/news/2021.md:34]: Thank you for investigating this. Remarklint/remark-cli packaging is really confusing.

File: docs/tools/ruby/rails-best-practices.md [hunk 5]
```diff
@@ -1,16 +1,14 @@
...
-sidebar_label: Rails Best Practices (deprecated)
 ---
 
 # Rails Best Practices
 
-:::caution
+:::danger
 
-We have **deprecated** the support of Rails Best Practices because of the lack of active development of the tool.
+We have dropped the support for Rails Best Practices on **September 30, 2021**.
```
Comment (suetsuguk) [inline docs/tools/ruby/rails-best-practices.md:10]:
```suggestion
We have dropped the support for Rails Best Practices on **October 4, 2021**.
```

File: docs/tools/others/actionlint.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (suetsuguk) [inline docs/tools/others/actionlint.md:1]: Delete this file just before merging.

**Review by dev-yakuza — APPROVED** [review]
LGTM 👍

**Comment by suetsuguk** [comment]
Thank you for taking your time.

**Pull request merged**